resources/assets/js/components/Profile.vue

```diff
@@ -801,7 +801,7 @@
 				return 'background-image: url(' + preview + ');';
 			},
 
-			blurhHashMedia(status) {
+			blurHashMedia(status) {
```

Comment:
This was added by @dansup in #2599 https://github.com/pixelfed/pixelfed/pull/2599/commits/39e389dd4e3221c6081fd2b9b5b521ac72c22c4a

I understand that `blurhash` is a thing, and I could imagine abbreviating it as `blurh`, but I can't see any sign of this function being used, and I'd expect not to write blur-hash-hash in general, unless one is trying to say "hash media using `blurhash`" in which case I'd personally write it as `blurhashHashMedia` ... Thinking about it a bit more, this should probably be:

```suggestion
			blurhashMedia(status) {
```